http4s-client/src/main/scala/com/avast/server/toolkit/http4s/Http4sClientConfig.scala

```diff
@@ -0,0 +1,30 @@
+package com.avast.server.toolkit.http4s
+
+import java.util.concurrent.TimeUnit
+
+import com.avast.server.toolkit.ssl.SslContextConfig
+import org.http4s.BuildInfo
+import org.http4s.client.blaze.ParserMode
+import org.http4s.client.defaults
+import org.http4s.headers.{`User-Agent`, AgentComment, AgentProduct}
+
+import scala.concurrent.duration.{Duration, FiniteDuration}
+
+final case class Http4sClientConfig(
+  responseHeaderTimeout: Duration = Duration.Inf,
+  idleTimeout: FiniteDuration = Duration(1, TimeUnit.MINUTES),
+  requestTimeout: FiniteDuration = defaults.RequestTimeout,
+  connectTimeout: FiniteDuration = defaults.ConnectTimeout,
+  userAgent: `User-Agent` = `User-Agent`(AgentProduct("http4s-blaze", Some(BuildInfo.version)), List(AgentComment("Server"))),
```

Comment:
I would prefer to remove `"-blaze"` substring. E.g. change the _agent_ to `http4s-client`.

Reply:
it's better to be explicit here -- we could have different `http4s` clients besides the "default" Blaze one

Reply:
The problem is the `"default"` :) I think that this configuration should be implementation-agnostic so `http4s-client` is better for me.

Another solution would be renaming the case-class to `BlazeHttp4sClientConfig`.

Reply:
`Http4sClientBlazeConfig` perhaps?

Reply:
I think the right name is `Http4sBlazeClientConfig` 😄 But I have good reasons. You want it to start with Http4s so that you can find it easily and the rest just copies the name of the artifact which is `http4s-blaze-client`.